custom/icds_reports/tests/__init__.py

```diff
@@ -110,9 +114,18 @@ def setUpModule():
 
         try:
             move_ucr_data_into_aggregation_tables(datetime(2017, 5, 28), intervals=2)
-        except AssertionError:
-            pass
-    _call_center_domain_mock.stop()
+        except AssertionError as e:
+            # we always use soft assert to email when the aggregation has completed
+            if "Aggregation completed" not in str(e):
```

Comment:
thank you! this will be so much nicer than trying to figure out what aggregation silently failed